scss/window/_layout.scss

```diff
@@ -21,19 +21,23 @@
 
     // Title bar
     .k-window-titlebar {
-        padding: $window-titlebar-padding-y $window-titlebar-padding-x;
-        min-height: 16px;
-        border-width: 0 0 1px;
-        border-style: solid;
+        padding: $window-titlebar-padding;
+        min-height: $window-titlebar-min-height;
+        border-width: $window-titlebar-border-width;
+        border-style: $window-titlebar-border-style;
         white-space: nowrap;
         display: flex;
         flex-direction: row;
         flex-shrink: 0;
         align-items: center;
     }
+
+    // Title bar title text
     .k-window-title {
         margin: 0;
-        font-size: $font-size-lg;
+        font-size: $window-title-font-size;
+        font-family: $window-title-font-family;
+        text-transform: $window-title-text-transform;
```

Comment:
@gyoshev 

We for the most part try to not override or create custom CSS - it becomes quite fragile and we have been burned by the Kendo theme changing between our releases. I prefer the ability to get this concept into the theme and for it to be supported as there is less CSS for us to maintain within our system. 

> do not allow to be used anywhere else.I can't think of elsewhere in the theme where there is a titlebar?

^ Is there an opportunity to change this? If a user declares a value of the variable it would be applied - otherwise it null and not included.

https://www.sassmeister.com/gist/2fc7f5c9bd25b6563ebaa70721273f62